### PR TITLE
RELEASE_NOTES: propose a few minor language tweaks

### PR DESCRIPTION
**Pull request by junkyardsparkle**
@aurelienpierre Can you comment on whether these changes are OK for you? I'm just trying to streamline some longer parts, hoping to increase the chances of all of this actually being read. :-)

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -105,22 +105,22 @@ making a backup is strongly advised.
...
 
 - A new module 'filmic RGB' which, like the previous 'filmic', is designed
-  to replace the 'base curve', 'shadows and highlights' modules and other tone-mapping options. 
+  to replace 'base curve', 'shadows and highlights' and other tone-mapping modules. 
   This new version replaces the one introduced in 2.6.2; it should be easier to use,
   and it will reduce color casts. The old 'filmic' module is now deprecated and
   is only available on images where it was already used for editing.
 
-- A new module 'tone equalizer', designed to merge the features of 'local tone mapping', 
+- A new module 'tone equalizer', designed to merge the features of 'tone mapping',
```
Comment (aurelienpierre) [inline RELEASE_NOTES.md]: the module is called "tone mapping", but it's actually a local tone mapping, so it should be put at least : 
(local) 'tone mapping'

Comment (junkyardsparkle) [reply RELEASE_NOTES.md]: Ok. About that distinction: should filmic rgb be said to replace other *global* tone mapping modules, or is that too specific in that case?

Comment (aurelienpierre) [reply RELEASE_NOTES.md]: Yes, indeed, filmic is a global tone mapping.

Comment (junkyardsparkle) [reply RELEASE_NOTES.md]: Done.

**Comment by TurboGit** [comment]
Thanks !

**Pull request merged**